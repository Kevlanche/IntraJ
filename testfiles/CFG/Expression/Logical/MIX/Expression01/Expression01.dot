digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[34]"; "Opt[35]"; "Exit[36]"}
{ rank = same; "List[15]"; "ParameterDeclaration[18]"; "ParameterDeclaration[22]"; "ParameterDeclaration[26]"; "ParameterDeclaration[30]"; "Block[36]"}
{ rank = same; "Modifiers[19]"; "PrimitiveTypeAccess[21]"; "Modifiers[23]"; "PrimitiveTypeAccess[25]"; "Modifiers[27]"; "PrimitiveTypeAccess[29]"; "Modifiers[31]"; "PrimitiveTypeAccess[33]"; "List[37]"}
{ rank = same; "List[20]"; "List[24]"; "List[28]"; "List[32]"; "VarDeclStmt[38]"; "ExprStmt[46]"}
{ rank = same; "Modifiers[39]"; "PrimitiveTypeAccess[41]"; "List[42]"; "AssignSimpleExpr[47]"}
{ rank = same; "List[40]"; "VariableDeclarator[43]"; "VarAccess[48]"; "OrLogicalExpr[49]"}
{ rank = same; "List[44]"; "Opt[45]"; "AndLogicalExpr[50]"; "AndLogicalExpr[61]"}
{ rank = same; "LogNotExpr[51]"; "LogNotExpr[56]"; "GTExpr[62]"; "GTExpr[65]"}
{ rank = same; "ParExpr[52]"; "ParExpr[57]"; "VarAccess[63]"; "IntegerLiteral[64]"; "VarAccess[66]"; "IntegerLiteral[67]"}
{ rank = same; "GTExpr[53]"; "GTExpr[58]"}
{ rank = same; "VarAccess[54]"; "IntegerLiteral[55]"; "VarAccess[59]"; "IntegerLiteral[60]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Expression01 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[18]" [label="ParameterDeclaration\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[19]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[20]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[21]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[22]" [label="ParameterDeclaration\n < b >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[23]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[25]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[26]" [label="ParameterDeclaration\n < c >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[27]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[28]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[29]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[30]" [label="ParameterDeclaration\n < d >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[31]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[33]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[35]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[36]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[37]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[38]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[39]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[40]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[41]" [label="PrimitiveTypeAccess\n < boolean >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[42]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[43]" [label="VariableDeclarator\n < boolean r >", fillcolor=white   style=filled  ]
"List[44]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[45]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[46]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[47]" [label="AssignSimpleExpr\n < r >", fillcolor=white   style=filled  ]
"VarAccess[48]" [label="VarAccess\n < r >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"OrLogicalExpr[49]" [label="OrLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AndLogicalExpr[50]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"LogNotExpr[51]" [label="LogNotExpr\n ", fillcolor=white   style=filled  ]
"ParExpr[52]" [label="ParExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[53]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[54]" [label="VarAccess\n < a >", fillcolor=white   style=filled  ]
"IntegerLiteral[55]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"LogNotExpr[56]" [label="LogNotExpr\n ", fillcolor=white   style=filled  ]
"ParExpr[57]" [label="ParExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[58]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[59]" [label="VarAccess\n < b >", fillcolor=white   style=filled  ]
"IntegerLiteral[60]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"AndLogicalExpr[61]" [label="AndLogicalExpr\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"GTExpr[62]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[63]" [label="VarAccess\n < c >", fillcolor=white   style=filled  ]
"IntegerLiteral[64]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"GTExpr[65]" [label="GTExpr\n ", fillcolor=white   style=filled  ]
"VarAccess[66]" [label="VarAccess\n < d >", fillcolor=white   style=filled  ]
"IntegerLiteral[67]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"Exit[36]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"Modifiers[19]" -> "List[20]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "Modifiers[19]" [style=dashed, color=gray]
"ParameterDeclaration[18]" -> "PrimitiveTypeAccess[21]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[18]" [style=dashed, color=gray]
"Modifiers[23]" -> "List[24]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "Modifiers[23]" [style=dashed, color=gray]
"ParameterDeclaration[22]" -> "PrimitiveTypeAccess[25]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[22]" [style=dashed, color=gray]
"Modifiers[27]" -> "List[28]" [style=dashed, color=gray]
"ParameterDeclaration[26]" -> "Modifiers[27]" [style=dashed, color=gray]
"ParameterDeclaration[26]" -> "PrimitiveTypeAccess[29]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[26]" [style=dashed, color=gray]
"Modifiers[31]" -> "List[32]" [style=dashed, color=gray]
"ParameterDeclaration[30]" -> "Modifiers[31]" [style=dashed, color=gray]
"ParameterDeclaration[30]" -> "PrimitiveTypeAccess[33]" [style=dashed, color=gray]
"List[17]" -> "ParameterDeclaration[30]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[34]" [style=dashed, color=gray]
"Modifiers[39]" -> "List[40]" [style=dashed, color=gray]
"VarDeclStmt[38]" -> "Modifiers[39]" [style=dashed, color=gray]
"VarDeclStmt[38]" -> "PrimitiveTypeAccess[41]" [style=dashed, color=gray]
"VariableDeclarator[43]" -> "List[44]" [style=dashed, color=gray]
"VariableDeclarator[43]" -> "Opt[45]" [style=dashed, color=gray]
"List[42]" -> "VariableDeclarator[43]" [style=dashed, color=gray]
"VarDeclStmt[38]" -> "List[42]" [style=dashed, color=gray]
"List[37]" -> "VarDeclStmt[38]" [style=dashed, color=gray]
"AssignSimpleExpr[47]" -> "VarAccess[48]" [style=dashed, color=gray]
"GTExpr[53]" -> "VarAccess[54]" [style=dashed, color=gray]
"GTExpr[53]" -> "IntegerLiteral[55]" [style=dashed, color=gray]
"ParExpr[52]" -> "GTExpr[53]" [style=dashed, color=gray]
"LogNotExpr[51]" -> "ParExpr[52]" [style=dashed, color=gray]
"AndLogicalExpr[50]" -> "LogNotExpr[51]" [style=dashed, color=gray]
"GTExpr[58]" -> "VarAccess[59]" [style=dashed, color=gray]
"GTExpr[58]" -> "IntegerLiteral[60]" [style=dashed, color=gray]
"ParExpr[57]" -> "GTExpr[58]" [style=dashed, color=gray]
"LogNotExpr[56]" -> "ParExpr[57]" [style=dashed, color=gray]
"AndLogicalExpr[50]" -> "LogNotExpr[56]" [style=dashed, color=gray]
"OrLogicalExpr[49]" -> "AndLogicalExpr[50]" [style=dashed, color=gray]
"GTExpr[62]" -> "VarAccess[63]" [style=dashed, color=gray]
"GTExpr[62]" -> "IntegerLiteral[64]" [style=dashed, color=gray]
"AndLogicalExpr[61]" -> "GTExpr[62]" [style=dashed, color=gray]
"GTExpr[65]" -> "VarAccess[66]" [style=dashed, color=gray]
"GTExpr[65]" -> "IntegerLiteral[67]" [style=dashed, color=gray]
"AndLogicalExpr[61]" -> "GTExpr[65]" [style=dashed, color=gray]
"OrLogicalExpr[49]" -> "AndLogicalExpr[61]" [style=dashed, color=gray]
"AssignSimpleExpr[47]" -> "OrLogicalExpr[49]" [style=dashed, color=gray]
"ExprStmt[46]" -> "AssignSimpleExpr[47]" [style=dashed, color=gray]
"List[37]" -> "ExprStmt[46]" [style=dashed, color=gray]
"Block[36]" -> "List[37]" [style=dashed, color=gray]
"Opt[35]" -> "Block[36]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[35]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[36]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[36]" -> "AssignSimpleExpr[47]" [color=red, constraint=false]
"AssignSimpleExpr[47]" -> "Exit[36]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[47]" -> "LogNotExpr[56]" [color=red, constraint=false]
"AssignSimpleExpr[47]" -> "GTExpr[62]" [color=red, constraint=false]
"AssignSimpleExpr[47]" -> "GTExpr[65]" [color=red, constraint=false]
"GTExpr[65]" -> "AssignSimpleExpr[47]" [color=blue, constraint=false, xlabel=" "] 
"GTExpr[65]" -> "IntegerLiteral[67]" [color=red, constraint=false]
"IntegerLiteral[67]" -> "GTExpr[65]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[67]" -> "VarAccess[66]" [color=red, constraint=false]
"VarAccess[66]" -> "IntegerLiteral[67]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[66]" -> "GTExpr[62]" [color=red, constraint=false]
"GTExpr[62]" -> "VarAccess[66]" [color=blue, constraint=false, xlabel="T "] 
"GTExpr[62]" -> "AssignSimpleExpr[47]" [color=blue, constraint=false, xlabel="F "] 
"GTExpr[62]" -> "IntegerLiteral[64]" [color=red, constraint=false]
"IntegerLiteral[64]" -> "GTExpr[62]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[64]" -> "VarAccess[63]" [color=red, constraint=false]
"VarAccess[63]" -> "IntegerLiteral[64]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[63]" -> "LogNotExpr[51]" [color=red, constraint=false]
"VarAccess[63]" -> "LogNotExpr[56]" [color=red, constraint=false]
"LogNotExpr[56]" -> "AssignSimpleExpr[47]" [color=blue, constraint=false, xlabel="T "] 
"LogNotExpr[56]" -> "VarAccess[63]" [color=blue, constraint=false, xlabel="F "] 
"LogNotExpr[56]" -> "GTExpr[58]" [color=red, constraint=false]
"GTExpr[58]" -> "LogNotExpr[56]" [color=blue, constraint=false, xlabel=" "] 
"GTExpr[58]" -> "IntegerLiteral[60]" [color=red, constraint=false]
"IntegerLiteral[60]" -> "GTExpr[58]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[60]" -> "VarAccess[59]" [color=red, constraint=false]
"VarAccess[59]" -> "IntegerLiteral[60]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[59]" -> "LogNotExpr[51]" [color=red, constraint=false]
"LogNotExpr[51]" -> "VarAccess[59]" [color=blue, constraint=false, xlabel="T "] 
"LogNotExpr[51]" -> "VarAccess[63]" [color=blue, constraint=false, xlabel="F "] 
"LogNotExpr[51]" -> "GTExpr[53]" [color=red, constraint=false]
"GTExpr[53]" -> "LogNotExpr[51]" [color=blue, constraint=false, xlabel=" "] 
"GTExpr[53]" -> "IntegerLiteral[55]" [color=red, constraint=false]
"IntegerLiteral[55]" -> "GTExpr[53]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[55]" -> "VarAccess[54]" [color=red, constraint=false]
"VarAccess[54]" -> "IntegerLiteral[55]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[54]" -> "VariableDeclarator[43]" [color=red, constraint=false]
"VariableDeclarator[43]" -> "VarAccess[54]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[43]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "VariableDeclarator[43]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[34]" -> "Opt[35]" -> "Exit[36]" [style=invis]
"ParameterDeclaration[18]" -> "ParameterDeclaration[22]" -> "ParameterDeclaration[26]" -> "ParameterDeclaration[30]" [style=invis]
"Modifiers[19]" -> "PrimitiveTypeAccess[21]" [style=invis]
"Modifiers[23]" -> "PrimitiveTypeAccess[25]" [style=invis]
"Modifiers[27]" -> "PrimitiveTypeAccess[29]" [style=invis]
"Modifiers[31]" -> "PrimitiveTypeAccess[33]" [style=invis]
"VarDeclStmt[38]" -> "ExprStmt[46]" [style=invis]
"Modifiers[39]" -> "PrimitiveTypeAccess[41]" -> "List[42]" [style=invis]
"List[44]" -> "Opt[45]" [style=invis]
"VarAccess[48]" -> "OrLogicalExpr[49]" [style=invis]
"AndLogicalExpr[50]" -> "AndLogicalExpr[61]" [style=invis]
"LogNotExpr[51]" -> "LogNotExpr[56]" [style=invis]
"VarAccess[54]" -> "IntegerLiteral[55]" [style=invis]
"VarAccess[59]" -> "IntegerLiteral[60]" [style=invis]
"GTExpr[62]" -> "GTExpr[65]" [style=invis]
"VarAccess[63]" -> "IntegerLiteral[64]" [style=invis]
"VarAccess[66]" -> "IntegerLiteral[67]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
69", fillcolor=1]
Edges [label="Total Nr. Edges
108", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
18", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
20", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}