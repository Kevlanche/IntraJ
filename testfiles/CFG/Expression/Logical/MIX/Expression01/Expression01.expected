NodeName: Entry[13] Successors: [VariableDeclarator[43]]
NodeName: VariableDeclarator[43] Successors: [VarAccess[54]]
NodeName: VarAccess[54] Successors: [IntegerLiteral[55]]
NodeName: IntegerLiteral[55] Successors: [GTExpr[53]]
NodeName: GTExpr[53] Successors: [LogNotExpr[51]]
NodeName: LogNotExpr[51] Successors: [VarAccess[59], VarAccess[63]]
NodeName: VarAccess[59] Successors: [IntegerLiteral[60]]
NodeName: IntegerLiteral[60] Successors: [GTExpr[58]]
NodeName: GTExpr[58] Successors: [LogNotExpr[56]]
NodeName: LogNotExpr[56] Successors: [AssignSimpleExpr[47], VarAccess[63]]
NodeName: AssignSimpleExpr[47] Successors: [Exit[36]]
NodeName: Exit[36] Successors: []
NodeName: VarAccess[63] Successors: [IntegerLiteral[64]]
NodeName: IntegerLiteral[64] Successors: [GTExpr[62]]
NodeName: GTExpr[62] Successors: [VarAccess[66], AssignSimpleExpr[47]]
NodeName: VarAccess[66] Successors: [IntegerLiteral[67]]
NodeName: IntegerLiteral[67] Successors: [GTExpr[65]]
NodeName: GTExpr[65] Successors: [AssignSimpleExpr[47]]

NodeName: Exit[36] Predecessor: [AssignSimpleExpr[47]]
NodeName: AssignSimpleExpr[47] Predecessor: [LogNotExpr[56], GTExpr[62], GTExpr[65]]
NodeName: LogNotExpr[56] Predecessor: [GTExpr[58]]
NodeName: GTExpr[58] Predecessor: [IntegerLiteral[60]]
NodeName: IntegerLiteral[60] Predecessor: [VarAccess[59]]
NodeName: VarAccess[59] Predecessor: [LogNotExpr[51]]
NodeName: LogNotExpr[51] Predecessor: [GTExpr[53]]
NodeName: GTExpr[53] Predecessor: [IntegerLiteral[55]]
NodeName: IntegerLiteral[55] Predecessor: [VarAccess[54]]
NodeName: VarAccess[54] Predecessor: [VariableDeclarator[43]]
NodeName: VariableDeclarator[43] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: GTExpr[62] Predecessor: [IntegerLiteral[64]]
NodeName: IntegerLiteral[64] Predecessor: [VarAccess[63]]
NodeName: VarAccess[63] Predecessor: [LogNotExpr[51], LogNotExpr[56]]
NodeName: GTExpr[65] Predecessor: [IntegerLiteral[67]]
NodeName: IntegerLiteral[67] Predecessor: [VarAccess[66]]
NodeName: VarAccess[66] Predecessor: [GTExpr[62]]
